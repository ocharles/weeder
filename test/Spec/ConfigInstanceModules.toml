root-instances = [ { module = "Spec.ConfigInstanceModules.Module1", instance = "Bounded T" }
                 , "Read T" 
                 , { module = "Spec.ConfigInstanceModules.Module3" }
                 , { class = '\.Enum$' }
                 , { module = "Spec.ConfigInstanceModules.Module2", class = '\.Show$' } 
                 ]
