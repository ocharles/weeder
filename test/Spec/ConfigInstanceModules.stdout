main: test/Spec/ConfigInstanceModules/Module1.hs:3:24: (Instance) :: Show T
main: test/Spec/ConfigInstanceModules/Module2.hs:3:30: (Instance) :: Bounded T
